.github/ISSUE_TEMPLATE/work-item.md

```diff
@@ -0,0 +1,53 @@
+# Work Item Issue Template
+description:
+title: "[Work_Item]"
+labels: ["work item"]
+assignees: [shawnalpay, jpradocueva]
+
+### **Template Usage Notes**:
+1. All fields marked as **mandatory** [*] must be filled before submission.  While some datapoints are optional for the initial creation of the work item, all datapoints must be provided in order to be considered for spec development.
+2. If you have suggestions for the specification but cannot fill out all fields in this issue template, please fill out a [Feedback] or [Discussion] issue template instead.
+3. For **Supporting Documentation**, ensure that linked files are accessible to relevant stakeholders.
+   
+---
+
+## 1. **Problem Statement** *
+Describe the problem, issue, use case, or opportunity that this work item addresses.
+Include practitioner quotes illustrating real examples a) of questions being asked by practitioners and b) value unlocked by answering these questions, if available.
+- **What is the problem?**: Explain the context and why it needs resolution.
+- **Impact**: Describe how the problem affects users, systems, or the project.
+
+Your input ...
+
+## 2. **Objective** *
+State the objective of this work item. What outcome is expected?
+- **Success Criteria**: Define how success will be measured (e.g. metrics and KPIs).
+
+Your input ...
+
+## 3. **Supporting Documentation** *
+Include links to supporting documents such as:
+- Data Examples: [Link to data or relevant files; DO NOT share proprietary information]
+- Related Use Cases or Discussion Documents: [Link to discussion]
+- PRs or Other References: [Link to relevant references]
+
+Your input ...
+
+## 4. **Proposed Solution / Approach**
```

Comment:
@jpradocueva  can you please confirm whether we are allowed to ask for and use solutions proposed by someone who hasn't signed the CLA?
Note: A similar section exists in the Discussion template as well.

Reply:
Per our discussion, I have added guidance in the following line.  Resolving.